### PR TITLE
Add to nodejs mic vad doco

### PR DESCRIPTION
**Pull request by irab**
Adding some more context for this nodejs example and some troubleshooting I needed to do to get it to work.

### CODE REVIEW

File: nodejs_mic_vad_streaming/Readme.md [hunk 1]
```diff
@@ -5,40 +5,72 @@ DeepSpeech with voice activity detection.
 
 ### Prerequisites:
 
-1) The example utilized the [mic](https://github.com/ashishbajaj99/mic) NPM module which requires
-either [sox](http://sox.sourceforge.net/) (Windows/Mac) or [arecord](http://alsa-project.org/) (Linux).
+Node LTS v14.14 installed:
```
Comment (lissyx) [inline nodejs_mic_vad_streaming/Readme.md:8]: That looks super-strict, can we have more details ? Is it what you tested to be successfull, or are there issues with other versions ?

File: nodejs_mic_vad_streaming/Readme.md [hunk 2]
```diff
@@ -5,40 +5,72 @@ DeepSpeech with voice activity detection.
...
+```bash
+nvm install 14.14
```
Comment (lissyx) [inline nodejs_mic_vad_streaming/Readme.md:10]: I would advise against documenting `nvm` out of hand like that. Either we give more steps / link to upstream doc, or we don't document with `nvm` at all.

File: nodejs_mic_vad_streaming/Readme.md [hunk 3]
```diff
@@ -5,40 +5,72 @@ DeepSpeech with voice activity detection.
...
-Is needed the library **libasound2-dev**
+1. [arecord](http://alsa-project.org/)
+1. The library **libasound2-dev**
+1. [mic](https://github.com/ashishbajaj99/mic) NPM module - which requires [sox](http://sox.sourceforge.net/)
+1. The pre-trained DeepSpeech english model and scorer(1089MB)
 
 ```
-$ sudo apt-get install libasound2-dev
+sudo apt-get install libasound2-dev alsa-utils
+npm install mic
```
Comment (lissyx) [inline nodejs_mic_vad_streaming/Readme.md:33]: Shoudn't that `mic` be a dependency set at the `package.json` level instead ?

**Review by lissyx — APPROVED** [review]
Looks good, but please fix the comments :)

File: nodejs_mic_vad_streaming/Readme.md (hunk 1 above)
Comment (irab) [reply nodejs_mic_vad_streaming/Readme.md:8]: That's just what I tested with, happy to change it to just 'tested with v14.14'. Not sure what versions it runs nicely with unfortunately and don't feel like testing all the versions 😅

File: nodejs_mic_vad_streaming/Readme.md (hunk 3 above)
Comment (irab) [reply nodejs_mic_vad_streaming/Readme.md:33]: Errgh, it looks like it already is.

**Comment by irab** [comment]
@lissyx I've updated as you suggested.

**Review by lissyx — APPROVED** [review]

**Pull request merged**